assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -11690,9 +11688,9 @@ private void initializeAssignment(SessionState state)
 			try {
 				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
 				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
-					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{contentreviewSiteMin, reviewServiceName}));
+					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
```

Comment:
If you access the whole file you can see this code:

if(allowReviewService && allowLTIReviewService){
			Site st = null;
			try {
				st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
				if (contentreviewSiteMin > 0 && st.getTitle().length() < contentreviewSiteMin){
					addAlert(state, rb.getFormattedMessage("review.sitechars", new Object[]{reviewServiceName, contentreviewSiteMin}));
				} else if (contentreviewSiteMax > 0 && st.getTitle().length() > contentreviewSiteMax){
					addAlert(state, rb.getFormattedMessage("review.sitecharslong", new Object[]{reviewServiceName, contentreviewSiteMax}));
				}
etc

You need to check it in there just to warn the user if the site title doesn't fit TII or CR requirements. Not sure where's the problem